server/commands/documentImporter.ts

```diff
@@ -143,42 +150,48 @@ async function confluenceToMarkdown(file): Promise<string> {
 }
 
 async function documentImporter({
-  file,
+  mimeType,
+  fileName,
+  content,
   user,
   ip,
+  transaction,
 }: {
   user: User;
-  file: File;
```

Comment:
It was necessary to change this command so that we could stop writing temporary files and just pass a `string` or a `Buffer` directly instead. As a bonus it allowed removing some `@ts-expect-error`.